tests/nested/manual/core20-to-core22/task.yaml

```diff
@@ -13,31 +13,34 @@ environment:
   # after remodel completes
   NESTED_ENABLE_TPM: false
   NESTED_ENABLE_SECURE_BOOT: false
-  NESTED_BUILD_SNAPD_FROM_CURRENT: false
```

Comment:
so the default value of NESTED_BUILD_SNAPD_FROM_CURRENT is true, meaning that the step in prepare about `repack_snapd_deb_into_snap` is now done automatically in the suite or each prepare right?

Reply:
@anonymouse64, right. the default for the repack for kernel gadget and core is true, and build_from_current for snapd is true as well. In this case just saying the repacks are not needed is enough.